stpsf/utils.py

```diff
@@ -205,7 +205,7 @@ def auto_download_stpsf_data():
 
             # Extract the tarball
             with tarfile.open(filename, "r:gz") as tar:
-                tar.extractall(default_path.parent, filter="fully_trusted")
+                tar.extractall(default_path.parent)
```

Comment:
This is a correct fix, as the default (for now) is "fully_trusted".  I think it would be prudent to provide a small comment that `# TODO filter="fully_trusted" will need to be added as a parameter once python 3.14 is adopted due to advertised changing defaults in python.`

Reply:
Sounds good, I just added the note